### ISSUE TITLE
alexa-remote-account SyntaxError

### ISSUE THREAD

**Issue by derekoharrow** [586837r/node-red-contrib-alexa-remote2#23]
Installing this and it errors during the install:
```
9 Jun 17:22:24 - [info]  - node-red-contrib-alexa-remote2:alexa-remote-account : SyntaxError: Unexpected token {,
9 Jun 17:22:24 - [info] Installed module: node-red-contrib-alexa-remote2,
9 Jun 17:22:24 - [info] Added node types:,
9 Jun 17:22:24 - [info]  - node-red-contrib-alexa-remote2:alexa-remote-init,
9 Jun 17:22:24 - [info]  - node-red-contrib-alexa-remote2:alexa-remote-get,
9 Jun 17:22:24 - [info]  - node-red-contrib-alexa-remote2:alexa-remote-command,
9 Jun 17:22:15 - [info] Installing module: node-red-contrib-alexa-remote2, version: 2.2.2,
9 Jun 17:22:24 - [info]  - node-red-contrib-alexa-remote2:alexa-remote-set,
9 Jun 17:22:24 - [info]  - node-red-contrib-alexa-remote2:alexa-remote-sequence,
9 Jun 17:22:24 - [info]  - node-red-contrib-alexa-remote2:alexa-remote-action,
9 Jun 17:22:24 - [info]  - node-red-contrib-alexa-remote2:alexa-remote-event
```

Also, when I try to use it, it keeps reporting errors about missing node alexa-remote-account, which I presume is because of this error.

**Comment by 8xiom** [comment]
Same here. Node red running within a container (docker) on a Raspi 3B+.

**Comment by 586837r** [comment]
This error did not occur in node v10 but in node v8. It's fixed now! (v2.2.3)
